docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -200,10 +196,21 @@ The enterprise components come in two plans: Pro and Premium.
 | [Keyboard navigation](/components/data-grid/accessibility/#keyboard-navigation)           |    ✅     |                 ✅                 |                     ✅                     |
 | [Localization](/components/data-grid/localization/)                                       |    ✅     |                 ✅                 |                     ✅                     |
 
-### License key installation
+### Evaluation (trial) licenses
 
-Once you purchase a license, you'll receive a license key.
-This key should be provided to the enterprise package to remove the watermark and
+You are [free to install](https://mui.com/store/legal/mui-x-eula/#evaluation-trial-licenses) and try `DataGridPro` as long as it is not used for development of a feature intended for production.
```

Comment:
I anticipate for what we explored in https://github.com/mui-org/mui-store/pull/81 (we reverted this PR) but still it's going to be the same URL, as far as I know.